sdk-manifests/ably-ruby.yaml

```diff
@@ -47,6 +47,7 @@ compliance:
     Authentication:
     Channel:
       Attach:
+      Encryption: # setOptions on existing channel instance is missing (i.e. it works but is inelegant, requiring soft-deprecated mechanism)
```

Comment:
> it works but is inelegant

Out of interest, what is inelegant about it?

Reply:
I've expanded the comment in https://github.com/ably/features/pull/30/commits/85d6775203a87ccbf5a8298d621de236243b4e2c

Closing this conversation on @lmars' behalf so that I can get this pull request merged cleanly.